code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -886,11 +897,63 @@ var/list/datum/absorbed_dna/hivemind_bank = list()
 	shield.creator = M
 	M.put_in_hands(shield)
 	playsound(loc, 'sound/effects/blobattack.ogg', 30, 1)
-	src.visible_message("<span class='warning'>The end of [M]\'s hand inflates rapidly, forming a huge shield-like mass!</span>",
+	src.visible_message("<span class='danger'>The end of [M]\'s hand inflates rapidly, forming a huge shield-like mass!</span>",
 							"<span class='warning'>We inflate our hand into a robust shield.</span>",
 							"<span class='warning'>You hear organic matter ripping and tearing!</span>")
 	gibs(src.loc)
 
+/mob/proc/horror_form()
+	set category = "Changeling"
+	set name = "Horror Form (40)"
+	set desc = "Tear apart your human disguise, revealing your true form."
+
+	var/datum/changeling/changeling = changeling_power(40,0,0)
+	if(!changeling)	return 0
+	src.mind.changeling.chem_charges -= 40
+
+	var/mob/living/M = src
+
+	M.visible_message("<span class='danger'>[src] writhes and contorts, their body expanding to inhuman proportions!</span>")

```

Comment:
Can you add the second argument to this one? It's what the user sees from their own perspective, so it'd be something like, `You begin to transform into your horror form!`